hack/deploy/prometheus.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: downstream-prometheus-operator
+  namespace: redhat-data-federation
+spec:
+  displayName: downstream-prometheus-operator
+  image: quay.io/dbindra/ose-prometheus-operator:4.10.0_catsrc
```

Comment:
Can you please make sure that on running the following command,
```
curl -H "Authorization: Bearer `cat /var/run/secrets/kubernetes.io/serviceaccount/token`" --cacert /var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt https://prometheus.redhat-data-federation.svc:9339/metrics
```
inside the `prometheus-k8s-0` pod (in `prometheus` container) in the `openshift-monitoring` namespace, the output is similar to the one below?
![image](https://user-images.githubusercontent.com/33557095/173035145-86dc8f74-a84e-4ae4-8145-c22bee9704ea.png)
***
Can you attach a screenshot of the output (that you are getting) as well? Thanks.

Reply:
@rexagod : yes or put is similar to this
![image](https://user-images.githubusercontent.com/79908956/173053919-fa90f0df-2565-4c3f-bd49-b549f55f6dfe.png)

Reply:
Awesome, LGTM!